tests/vale/X64.Vale.Decls.fst

```diff
@@ -250,7 +252,9 @@ let add_wrap_lemma (x y:UInt64.t)
   (ensures add_wrap (UInt64.v x) (UInt64.v y) == UInt64.v (S.add_mod64 x y))
   [SMTPat (S.add_mod64 x y)]
 = ()
+#restart-solver
 irreducible let va_irreducible_lemma_Add64Wrap va_b0 va_s0 va_sN dst src =
+  admit(); //this succeeds, but takes a very long time, needlessly bloating CI
```

Comment:
I added two admits in this test file. These queries succeed both in master and here after several retries, but take a couple of hundred seconds. Not sure we're gaining much by keeping them in the test suite, especially since they're also not representative of how the Vale repo actually works.

Reply:
I agree and wouldn't mind removing these tests.